### ISSUE TITLE
Field of raw text

### ISSUE THREAD

**Issue by raheelqader** [pytorch/text#442]
I need a field to store my source text as as raw text. Is there a way to achieve this?

I tried to use RawField but it fives met this error: AttributeError: 'RawField' object has no attribute 'is_target'. I think #288 is causing the error.

**Comment by andreazampieri** [comment]
Hi!
I just encountered this problem myself, and for now I came up with this workaround [it's very unelegant but it should do the work]

```
class my_dataset():
  def __init__(self):
    self.RAW = data.RawField()
    self.RAW.is_target = False     #this is the workaround
    self.TEXT = data.Field(batch_first=True,
                           lower=True)
    self.LABEL = data.Field(sequential=False,
                            unk_token=None)


    self.train, self.dev, self.test = data.TabularDataset.splits(
        path='your/path/',
        train='train.tsv',
        validation='dev.tsv',
        test='test.tsv',
        format='tsv',
        fields=[('raw_field', self.RAW),
                ('text_field', self.TEXT),
                ('label_field',self.LABEL)])
```

Basically i modify on the fly the RawField adding the field that's missing

**Comment by mttk** [comment]
This has been fixed with #459, thanks for reporting!
